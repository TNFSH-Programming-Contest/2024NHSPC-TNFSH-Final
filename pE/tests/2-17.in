1 1678941 2605112 9616577 6974967
81
