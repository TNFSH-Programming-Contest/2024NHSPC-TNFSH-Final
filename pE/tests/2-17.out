54727887139309572
